docs/main-concepts/how-to-define-a-fetch.md

```diff
@@ -0,0 +1,352 @@
+---
+id: how-to-define-a-fetch
+title: How to define a fetch
+sidebar_label: How to define a fetch
+---
+
+The `defineFetch` API has the following shape:
+
+`makePersonFetch.js`
+
+```js
+import { defineFetch } from 'resift';
+
+const makePersonFetch = defineFetch({
+  displayName: /* {display name} */,
+  make: (/* {make args} */) => ({
+    key: /* {key} */,
+    request: (/* {request args} */) => ({ http }) => {
+      /* {request body} */
+    },
+  }),
+});
+
+export default makePersonFetch;
+```
+
+There is:
+
+- the `displayName`,
+- the `make` function and its arguments
+  - the `key` array
+  - the `request` function and its arguments, and
+    - the request body
+
+The following sections will go over how to fill out this shape.
+
+---
```

Comment:
I'd add a header here like

### Arguments
or
### Options

Guess options is more accurate since they aren't function args